### ISSUE TITLE
Cannot grunt build app without running into errors

### ISSUE THREAD

**Issue by georaldc** [yeoman/generator-webapp_DEPRECATED#632]
Was testing this to build a simple site and after installing and running the generator (which ran fine without any errors), grunt build would abort, probably due to the following:

```
Running "useminPrepare:html" (useminPrepare) task
Warning: An error occurred while processing a template (Unexpected token )). Use --force to continue.
```

**Comment by georaldc** [comment]
Oh, I guess adding the --force flag makes it complete the build process, but it runs into another warning on the way:

```
Running "imagemin:dist" (imagemin) task
Warning: An error occurred while processing a template (Unexpected token )). Used --force, continuing.
```

**Comment by stefanmeschke** [comment]
I think the problem is that the `useminPrepare:html` task is done after cleaning the `dist` directory and there is no html file to prepare. Same problem on the `imagemin:dist` task: no images the error occured?

**Comment by terrencekeane** [comment]
I was getting the same error and found that the gulp version of this generator, generator-gulp-webapp, doesn't seem to have the error. I just switched over to using that generator for the project I was starting. Not a solution but figured it wouldn't hurt to mention it here.

**Comment by tsck** [comment]
I believe the problem was that there were extra '%' characters added to the closing tags in the imagemin task.

Was:

``` json
cwd: '<%%= config.app %%>',
src: '*.{ico,png}',
dest: '<%%= config.dist %%>'
```

Fixed:

``` json
cwd: '<%%= config.app %>',
src: '*.{ico,png}',
dest: '<%%= config.dist %>'
```

I issued a pull request as per the above, just wanted to post it here for anyone having issues with this. Just delete those extra '%' characters and the build task should work.

**Comment by RonanCodes** [comment]
Brilliant, it's working now.
The Yo Webapp setup still contains the error.

**Comment by silvenon** [comment]
Released in [v1.1.2](https://github.com/yeoman/generator-webapp/releases/tag/v1.1.2).
